dependencies/influxdb/internal/service.go

```diff
@@ -0,0 +1,155 @@
+package internal
+
+// TODO: Remove this when https://github.com/influxdata/influxdb-client-go/pull/256 is merged
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"mime"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	http2 "github.com/influxdata/influxdb-client-go/v2/api/http"
```

Comment:
Can you rename this to something like `influxhttp`, `ihttp`, `httpapi`, or `apihttp`? I'd like a name other than `http2` that is a bit more descriptive.